public/app/features/integrated-alerting/components/AlertRuleTemplate/__mocks__/alertRuleTemplateStubs.ts

```diff
@@ -0,0 +1,22 @@
+import { Template, FormattedTemplate } from '../AlertRuleTemplatesTable/AlertRuleTemplatesTable.types';
```

Comment:
Change the base branch to `PMM-6942-alert-rule-template-edit` please because this file already exists there

Reply:
Done :)